control-base/src/main/java/fi/nls/oskari/control/layer/SaveLayerHandler.java

```diff
@@ -270,9 +268,18 @@ private boolean handleRequestToMapLayer(final ActionParameters params, OskariLay
             }
         }
 
-        // FIXME: Update JSON key when frontend implementation was updated
-        MaplayerGroup maplayerGroup = oskariMapLayerGroupService.find(params.getHttpParam("inspireTheme", -1));
-        ml.addGroup(maplayerGroup);
+        String groupId = params.getHttpParam("maplayerGroups", "-1");
```

Comment:
Could consider placing the `"maplayerGroups"` into a static final String PARAM_MAPLAYER_GROUPS. However SaveLayerHandler seems to use "undocumented" keys all over the place. On the other hand that's a poor excuse to not do something "the right way" :)